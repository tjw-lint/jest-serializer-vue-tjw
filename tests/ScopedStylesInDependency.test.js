const helpers = require('./helpers.js');

import { mount } from '@vue/test-utils';
import ScopedStylesInDependency from './components/ScopedStylesInDependency.vue';

describe('ScopedStylesInDependency.vue', () => {
  test('Snapshot data-v ids removed', () => {
    helpers.mockSettings({
      removeDataVId: true,
      removeComments: true
    });

    const wrapper = mount(ScopedStylesInDependency);

    expect(wrapper.html())
      .toMatchSnapshot();
  });

  test('Snapshot data-v ids remain', () => {
    helpers.mockSettings({
      removeDataVId: false,
      removeComments: true
    });

    const wrapper = mount(ScopedStylesInDependency);

    expect(wrapper)
      .toMatchSnapshot();
  });
});
