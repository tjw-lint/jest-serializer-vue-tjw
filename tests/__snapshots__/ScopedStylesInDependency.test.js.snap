// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScopedStylesInDependency.vue Snapshot data-v ids remain 1`] = `
<div>
  <div class="vgt-wrap ">
    <div class="vgt-inner-wrap">
      <div class="vgt-fixed-header">
      </div>
      <div class="vgt-responsive" style="overflow: scroll-y; max-height: auto;">
        <table class="vgt-table bordered ">
          <thead data-v-eb5a915e>
            <tr data-v-eb5a915e>
              <th data-v-eb5a915e class="vgt-left-align sortable" style="min-width: auto; max-width: auto; width: auto;">
                <span data-v-eb5a915e>Name</span>
              </th>
              <th data-v-eb5a915e class="vgt-left-align sortable" style="min-width: auto; max-width: auto; width: auto;">
                <span data-v-eb5a915e>Favorite data-v-id=""</span>
              </th>
            </tr>
          </thead>
          <tbody>
            <tr class>
              <td class="vgt-left-align">
                <span>
                  Dave
                </span>
              </td>
              <td class="vgt-left-align">
                <span>
                  data-v-13ae5fa3=""
                </span>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>
</div>
`;

exports[`ScopedStylesInDependency.vue Snapshot data-v ids removed 1`] = `
<div>
  <div class="vgt-wrap ">
    <div class="vgt-inner-wrap">
      <div class="vgt-fixed-header">
      </div>
      <div class="vgt-responsive" style="overflow: scroll-y; max-height: auto;">
        <table class="vgt-table bordered ">
          <thead>
            <tr>
              <th class="vgt-left-align sortable" style="min-width: auto; max-width: auto; width: auto;">
                <span>Name</span>
              </th>
              <th class="vgt-left-align sortable" style="min-width: auto; max-width: auto; width: auto;">
                <span>Favorite data-v-id=""</span>
              </th>
            </tr>
          </thead>
          <tbody>
            <tr class>
              <td class="vgt-left-align">
                <span>
                  Dave
                </span>
              </td>
              <td class="vgt-left-align">
                <span>
                  data-v-13ae5fa3=""
                </span>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>
</div>
`;
